### PR TITLE
Add SQL module description

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md [hunk 1]
```diff
@@ -478,7 +478,32 @@ public function main() {
 
 ```
 ## Standard Library
-### Introduced new JDBC module
+### Introduced new SQL module
+
+The newly introduced `sql` module provides a common interface and functionality to interact with a database. The corresponding database clients can be created by using specific database modules such as MySQL or using the Java Database Connectivity module JDBC.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md:483]:
```suggestion
The newly-introduced `sql` module provides a common interface and functionality to interact with a database. The corresponding database clients can be created by using specific database modules such as MySQL or using the Java Database Connectivity module JDBC. 
```

File: downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md [hunk 2]
```diff
@@ -478,7 +478,32 @@ public function main() {
 
 ```
 ## Standard Library
-### Introduced new JDBC module
+### Introduced new SQL module
+
+The newly introduced `sql` module provides a common interface and functionality to interact with a database. The corresponding database clients can be created by using specific database modules such as MySQL or using the Java Database Connectivity module JDBC. 
+
+The revamped SQL implementation has the support for `sql:ParameterizedQuery`, through which parameterized queries can be passed easily.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md:485]:
```suggestion
The revamped SQL implementation has the support for `sql:ParameterizedQuery` through which parameterized queries can be passed easily.
```

File: downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md [hunk 3]
```diff
@@ -478,7 +478,32 @@ public function main() {
...
+
+A sample connector for MySQL database is as follows,
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md:487]:
```suggestion
A sample connector for a MySQL database is as follows.
```

File: downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md [hunk 4]
```diff
@@ -516,8 +516,32 @@ public function main() {
...
+A sample connector for a MySQL database is as follows.
+```ballerina
+import ballerina/io;
+import ballerina/mysql;
+import ballerina/sql;
+
+public function main() returns sql:Error? {
+
+    mysql:Client mysqlClient = check new ("localhost", "root", "root", "testdb");
+    io:println("MySQL client with user and password created.");
```
Comment (daneshk) [inline downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md:534]:
```suggestion
```

Comment (niveathika) [reply downloads/swan-lake-release-notes/swan-lake-preview1/RELEASE_NOTE.md]: Fixed in 9fdbaf3

**Pull request merged**